Source/ProgressionSystemRuntime/Private/Components/PSHUDComponent.cpp

```diff
@@ -0,0 +1,302 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#include "Components/PSHUDComponent.h"
+//---
+#include "Bomber.h"
+#include "Controllers/MyPlayerController.h"
+#include "UtilityLibraries/MyBlueprintFunctionLibrary.h"
+#include "Data/PSDataAsset.h"
+#include "ProgressionSystemRuntimeModule.h"
+#include "Blueprint/WidgetTree.h"
+#include "UI/MyHUD.h"
+#include "Widgets/PSMenuWidget.h"
+#include "Data/PSTypes.h"
+#include "Data/PSSaveGameData.h"
+#include "Data/PSWorldSubsystem.h"
+#include "Components/Image.h"
+#include "GameFramework/MyGameStateBase.h"
+#include "GameFramework/MyPlayerState.h"
+#include "Kismet/GameplayStatics.h"
+#include "MyDataTable/MyDataTable.h"
+#include "MyUtilsLibraries/WidgetUtilsLibrary.h"
+
+#include UE_INLINE_GENERATED_CPP_BY_NAME(PSHUDComponent)
+
+// Sets default values for this component's properties
+UPSHUDComponent::UPSHUDComponent()
+{
+	// Set this component to be initialized when the game starts, and to be ticked every frame.  You can turn these features
+	// off to improve performance if you don't need them.
+	PrimaryComponentTick.bCanEverTick = false;
+	PrimaryComponentTick.bStartWithTickEnabled = false;
+}
+
+// Called when the game starts
+void UPSHUDComponent::BeginPlay()
+{
+	Super::BeginPlay();
+
+	UE_LOG(LogProgressionSystem, Warning, TEXT("--- I'm running ---"));
```

Comment:
Please remove this UE_LOG as not useful anymore

Reply:
removed